examples/modules/chess/components/checkerboard.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+const NUM_COLS = 8;
+const NUM_ROWS = 8;
+const PRIMARY_COLOR = '#d18b47';
+const SECONDARY_COLOR = '#ffce9e';
+const MOVABLE_COLOR = 'palegreen';
+const SELECTED_COLOR = 'green';
+const COL_NAMES = 'abcdefgh';
+
+class Checkerboard extends React.Component {
```

Comment:
This class assumes knowledge of the internals of the `Grid` implementation (`viewBox` specifically). It will break if we change the implementation from SVG to something else.

What do you think about just adding some props to the `Grid` class to do some of the color highlighting you need for the checkerboard (and getting rid of this class entirely)? I think they are general enough that they will be useful to a wide variety of games.